stdlib/public/TensorFlow/Ops.swift

```diff
@@ -53,7 +53,7 @@ public extension Tensor where Scalar : Numeric {
   /// Perform an element-wise type conversion from a `Bool` tensor.
   @_inlineable @inline(__always)
   init(_ other: Tensor<Bool>) {
-    self.init(handle: #tfop("Cast", other.handle, DstT: Scalar.self))
+    self = Raw.cast(other)
```

Comment:
Great to see arg labels are gone for trivial positional args!

Reply:
Indeed :) The trivial label list can be tweaked [here](https://github.com/tensorflow/swift-bindings/blob/master/generate_wrappers.py#L132) if needed btw.